### PR TITLE
Move over km command, update imports, add licensing, add km to build targets.

### PR DESCRIPTION
**Pull request by spacejam**
@jdef @tsenart

### CODE REVIEW

File: plugin/contrib/mesos/cmd/km/km.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+//TODO(jdef) cloned from k8s cmd/hypercube: this is a binary that can morph
+//into all of the other kubernetes-mesos binaries
+package main
+
+import (
+	"os"
+)
+
+func main() {
```
Comment (tsenart) [inline plugin/contrib/mesos/cmd/km/km.go:25]: Do we wanna set GOMAXPROCS here too? `runtime.GOMAXPROCS(runtime.NumCPU())`

**Comment by tsenart** [comment]
LGTM

File: plugin/contrib/mesos/cmd/km/km.go (hunk 1 above)
Comment (jdef) [inline plugin/contrib/mesos/cmd/km/km.go:25]: RunToExit sets GOMAXPROCS, probably redundant to do it here

**Comment by jdef** [comment]
LGTM

**Comment by jdef** [comment]
:ship:

**Pull request merged**